src/coreclr/jit/gentree.cpp

```diff
@@ -252,6 +252,7 @@ void GenTree::InitNodeSize()
     }
 
     GenTree::s_gtNodeSizes[GT_CALL]          = TREE_NODE_SZ_LARGE;
+    GenTree::s_gtNodeSizes[GT_CNS_VEC]       = TREE_NODE_SZ_LARGE;
```

Comment:
This can and should be a small node on 64 bit hosts.

Reply:
Fixed.